### PR TITLE
Add new package: py-acme-tiny

### PR DESCRIPTION
**Pull request by darmac**

### CODE REVIEW

File: var/spack/repos/builtin/packages/py-acme-tiny/package.py [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+from spack import *
+
+
+class PyAcmeTiny(PythonPackage):
+    """A tiny script to issue and renew TLS certs from Let's Encrypt."""
+
+    homepage = "https://github.com/diafygi/acme-tiny"
+    git      = "https://github.com/diafygi/acme-tiny.git"
+
+    version('master', branch='master')
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/py-acme-tiny/package.py:15]: Can you add a stable release?

File: var/spack/repos/builtin/packages/py-acme-tiny/package.py [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+    depends_on('py-setuptools-scm', type=('build', 'run'))
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/py-acme-tiny/package.py]: I imagine this is only needed at build time. Also missing dep on setuptools.

**Review by adamjstewart — CHANGES_REQUESTED** [review]

File: var/spack/repos/builtin/packages/py-acme-tiny/package.py (hunk 2 above)
Comment (darmac) [reply var/spack/repos/builtin/packages/py-acme-tiny/package.py]: If there is no setuptools at runtime, some error will report like that:
```
[root@localhost ~]# spack load py-acme-tiny
[root@localhost ~]# acme-tiny
Traceback (most recent call last):
  File "/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/py-acme-tiny-4.1.0-kfuxwmxaqjpflfdyitllt4gpio6zmnnm/bin/acme-tiny", line 10, in <module>
    from importlib.metadata import distribution
ModuleNotFoundError: No module named 'importlib.metadata'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/py-acme-tiny-4.1.0-kfuxwmxaqjpflfdyitllt4gpio6zmnnm/bin/acme-tiny", line 13, in <module>
    from importlib_metadata import distribution
ModuleNotFoundError: No module named 'importlib_metadata'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/py-acme-tiny-4.1.0-kfuxwmxaqjpflfdyitllt4gpio6zmnnm/bin/acme-tiny", line 15, in <module>
    from pkg_resources import load_entry_point
ModuleNotFoundError: No module named 'pkg_resources'
```

**Review by adamjstewart — APPROVED** [review]

**Comment by adamjstewart** [comment]
Can you rebase/merge develop to fix the vermin tests?

**Review by alalazo — APPROVED** [review]

**Pull request merged**